calc.js

```diff
@@ -0,0 +1,110 @@
+
+spinnerHide();
+
+function fiboN(n) {
+  let a = 0, b = 1, c, i;
+  if (n == 0)
+    return a;
+  for (i = 2; i <= n; i++) {
+    c = a + b;
+    a = b;
+    b = c;
+  }
+  return b;
+}
+
+function validateToCalc() {
+  let x = parseInt(document.getElementById('inputN').value);
+  document.getElementById('fiboResp').innerHTML = '';
+  document.getElementById('hideSpin').style.visibility = 'visible';
+  let saveCheckDefault = document.querySelector("#flexCheckDefault");
+  if (x && saveCheckDefault.checked) {
+    return callServer(x);
+  }
+  if (x > 50) {
+    spinnerHide();
+    return document.getElementById('fiboResp').innerHTML = `<div class="alert alert-danger" role="alert">Can't be larger than 50</div>`;
+  }
+  if (x < 1) {
+    spinnerHide();
+    document.getElementById('fiboResp').style.color = '#373A3C';
+    return document.getElementById('fiboResp').innerHTML = "Number can't be smaller than 1";
+  }
+  fiboCalc(x);
+}
+
+function fiboCalc(x) {
+  let y = fiboN(x);
```

Comment:
y is not a good name for variable. Variables & functions names should reflect the purpuse/usage/idea of them and help the reader of the codes understand it.